salt/etcd-proxy/init.sls

```diff
@@ -5,6 +5,10 @@ include:
   file.managed:
     - source: salt://etcd-proxy/etcd.conf
     - makedirs: True
+  module.run:
+    - name: service.systemctl_reload
+    - onchanges:
+      - file: /etc/systemd/system/etcd.service.d/etcd.conf
```

Comment:
I think we should move this code to `/salt/etcd/init.sls`. This `init.sls` inherits from there and just overwrites some details (like the config file path).

Reply:
Agree, however, that's unrelated to ensuring systemd is reloaded when we change the unit files... There was another (abandoned) PR to do that.. I'll revive it.

Reply:
PR to combine the roles: https://github.com/kubic-project/salt/pull/215